### PR TITLE
Upgrade to scalafix v0.9.0

### PR DESCRIPTION
**Pull request by olafurpg**
Followup of #156

### CODE REVIEW

**Comment by julienrf** [comment]
Does that mean that the changes proposed in #155 are not necessary?

**Comment by olafurpg** [comment]
I had forgotten about that one, I will incorporate those changes into this PR next week 👍

**Comment by olafurpg** [comment]
Incorporated the changes from #155 and addressed the remaining comment. This should be ready to go 💪

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To use this library, add the following to your build.sbt:
 libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "0.2.1"
 ```
 
-Version 0.2.1 is compatible with Scala 2.13.0-M5. For Scala 2.13.0-M4 you should use version 0.1.1.
+Version 0.2.1 is compatible with Scala
```
[the remark this answers is not in the document]
Comment (olafurpg) [reply README.md]: I blindly cherry-picked all commits from #155, added it back now.

File: README.md [hunk 2]
```diff
@@ -38,45 +38,56 @@ The library also adds backported versions of new collection types, currently `sc
...
-
-* `Collection213Upgrade` For upgrading applications (like web server, etc) from 2.11/2.12 to 2.13
-* `Collection213CrossCompat` For library that wants to cross compile to 2.11, 2.12 and 2.13
+The migration rules use scalafix. Please see the [official installation instruction](https://scalacenter.github.io/scalafix/docs/users/installation.html) and, in particular, check that your full Scala version is supported (ex 2.12.6).
 
 ```scala
 // project/plugins.sbt
 
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.7.0-RC1")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.0-RC1")
```
[the remark this answers is not in the document]
Comment (olafurpg) [reply README.md]: Good catch! Updated to 0.9.0

**Comment by julienrf** [comment]
That looks good, thank you so much!

**Pull request merged**